lib/std/simd.zig

```diff
@@ -381,7 +393,7 @@ test "vector prefix scan" {
     try std.testing.expectEqual(Vector(4, i32){ 11, 11, 9, -21 }, prefixScan(.Min, 1, int_base));
     try std.testing.expectEqual(Vector(4, i32){ 11, 23, 23, 23 }, prefixScan(.Max, 1, int_base));
 
-    // Not testing add and mul because I'm not gonna try predicting those rounding errors.
+    // Trying to predict the rounding errors of addition and multiplication of floats would be a mess, so we don't test those.
```

Comment:
Error range prediction is not that hard, if the arithmetic is simple. The hard part is to get the error ranges for each SIMD instruction + have a standard way to extract the user-set error range (see https://composter.com.ua/documents/AVX512.pdf or https://developer.apple.com/documentation/accelerate/simd_float4).
IEE754 has yet no standard that includes the NaN behavior, ie denormals and +-infty https://www.agner.org/optimize/nan_propagation.pdf so the programmer must ensure this does not happen or handle it on a per hardware case.

Reply:
Often SIMD does not conform IEE754 at all for performance.

Reply:
Are you telling me to change the comment or to implement the test?

Reply:
nope. this is just missing a reason why its a mess and what we could (not) do about it.